### PR TITLE
Docker compose without TLS #none

### LINKED ISSUE #326: Add docker-compose without TLS
Currently the `dev_utils` has TLS enabled by default, we need a docker-compose for local development to start the services, sda-db and sda-mq without TLS.

### PR DESCRIPTION
**Pull request by jbygdell**
Addition of a Docker Compose setup with TLS disabled including a simple integration test

closes #326

### CODE REVIEW

**Review by blankdots — COMMENTED** [review]
is there any way to add another strategy to https://github.com/neicnordic/sda-pipeline/blob/master/.github/workflows/integration.yml for no-tls and maybe some `README.md` instructions - pretty please

**Review by blankdots — APPROVED** [review]

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        storagetype: [s3, posix]
+        storagetype: [s3, posix, s3notls,]
```
Comment (blankdots) [inline .github/workflows/integration.yml:15]: forgotten comma ?
```suggestion
        storagetype: [s3, posix, s3notls]
```

**Review by blankdots — CHANGES_REQUESTED** [review]

**Review by blankdots — APPROVED** [review]
meant to approve it, that comma I am fine if it is there :)

**Review by norling — APPROVED** [review]
Can we use the compose v2 syntax, just to be on the newer version? Otherwise - looks good!

**Review by blankdots — COMMENTED** [review]
just noticed this: are these lines needed any more https://github.com/neicnordic/sda-pipeline/pull/327/files#diff-d4426dbe2254f7946e706886a3f294a6d57e1098c1ddb2e9a056f510ada5ca2cR25-R27 ?

https://github.com/neicnordic/sda-pipeline/blob/master/dev_utils/run_tests.sh and this ?

**Review by a-ghanem — APPROVED** [review]
Cannot say for sure as Norway is not using `docker-compose` in our deployment.

**Comment by blankdots** [comment]
all positive reviews, merging

**Pull request merged**